jxl/src/render/stages/from_linear.rs

```diff
@@ -106,12 +101,30 @@ impl RenderPipelineInPlaceStage for FromLinearStage {
             }
             TransferFunction::Gamma(g) => {
                 for row in row {
-                    for v in &mut row[..xsize] {
-                        *v = crate::util::fast_powf(v.abs(), g).copysign(*v)
+                    for x in (0..xsize).step_by(D::F32Vec::LEN) {
```

Comment:
probably better to use `.chunks_exact_mut(D::F32Vec::LEN).take(xsize.div_ceil(D::F32Vec::LEN))` (similar to Gaborish)